_posts/2017-05-17-ola.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Ola
+sourceURL: https://developers.google.com/web/showcase/2017/ola
+image: /images/ola.png
+tags:
+- App Shell
+- Home Screen
+- IndexedDB
+- Service Worker
+- Travel
+
+---
+
+Ola reports **30% higher conversion** than native for its PWA in Tier 3 cities and comparable rates in Tier 2 cities. Overall, Ola has seen a **68% increase** in mobile traffic and **20%** of users who book via the PWA had previously uninstalled the native app.
```

Comment:
Is there any way of replacing "Tier 3" and "Tier 2" with some sort of population numbers or something so the figure has better context? Alternatively I'd recommend putting those phrases in quotation marks (unless they're general knowledge to everyone but me, in which case please disregard).